rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,134 @@
+- [![done](https://img.shields.io/badge/convergence-done-green)](https://github.com/microsoft/fluentui/wiki/Package-status)
+
+Notes:
+
+- Charting and maybe experiments would probably fall into the "preview (legacy)" category
+- Suite packages should list which components are converged or not
+
+### Add `@fluentui/react-preview` and related communication channels
+
+Sometime in the next couple months after version 8 release, add a `@fluentui/react-preview` package which exports _only_ converged components which have reached "80% ready"/preview status. Notes:
+
+- Exports will go away once they're moved to the suite
+- Document clearly from the start that the stability is lower, and people can consume it if they're willing to accept that minor versions may sometimes contain breaking changes (to be clearly noted in the changelogs)
+- Either 1st or 3rd parties can use this if they're willing to accept the extra risk/work
+- Readme should be kept updated with status of each component
+- If people prefer, they can use the individual preview component packages, but `@fluentui/react-preview` is the "one-stop shop" for previewing converged components
+
+We should also make a private Teams channel within Fluent Community for internal partners who are working closely with us to try out the preview components. (It's possible that this could be expanded to external partners in the future.) This would be a very direct way for partners to share feedback and get support with preview components.
+
+### Related code moves
+
+Motivation: improve contribution story, in some ways improve consumption story.
+
+- All current v8 component implementations should go back into `react-internal`:
+  - Checkbox
+  - Link
+  - Pivot
+  - Slider
+  - Toggle
+  - All components which were put in `react` to resolve cycles
+- Move "converged" components into the package roots and get rid of `/next` folders.
+  - The component packages can depend on `react` if needed for shared implementation parts, at least temporarily.
+  - Before release, any deps on `react` would need to be removed and handled with slots intead: e.g. if we haven't gotten to converge Label yet, converged Checkbox by default has built-in `<label>` as its `label` slot. But `@fluentui/react` will define and export a version of Checkbox which uses the old Label.
+- _(done)_ ~~Delete react-next: it's not being used, causing minor confusion, and can come back later without too much trouble.~~
+- **Open question** (see below): should we merge `react-internal` back into `react`?
+
+## Discarded Solutions
+
+<!-- As you enumerate possible solutions, try to keep track of the discarded ones. This should include why we discarded the solution. -->
+
+## Open Issues
+
+<!-- Optional section, but useful for first drafts. Use this section to track open issues on unanswered questions regarding the design or proposal.  -->
+
+### Should `react-internal` move back into `react`?
```

Comment:
If we move everything back into `react` will this cause issues when we want to fold a converged component into `react` at some future point? 

I assume a converged component would replace the legacy component in `react`, but won't we need to support both for some time as partners slowly migrate?

Reply:
Ok, answered my own question by reading further.
Looks like the answer is "Legacy versions will move to `/compat` if still needed". 

I guess the main question here is, I thought we were trying to go away from path based imports (unfortunately I don't have reasoning around *why* we are going away from file based imports on hand).

Perhaps the bigger question here on moving `react-internal` back into `react` is: 
Should we hold off on making large not-quite necessary changes now, things are moving and churning a lot as we get going on converged components. I would hate to spend a bunch of time getting rid of `react-internal` only to realize in 2 months that we need it back.